editor/src/clj/editor/gui.clj

```diff
@@ -2522,19 +2525,20 @@
    (attach-texture self textures-node texture false))
   ([self textures-node texture internal?]
    (concat
-    (g/connect texture :_node-id self :nodes)
-    (g/connect texture :texture-gpu-textures self :texture-gpu-textures)
-    (g/connect texture :texture-infos self :own-texture-infos)
-    (when (not internal?)
-      (concat
-       (g/connect texture :dep-build-targets self :dep-build-targets)
-       (g/connect texture :pb-msg self :texture-msgs)
-       (g/connect texture :build-errors textures-node :build-errors)
-       (g/connect texture :node-outline textures-node :child-outlines)
-       (g/connect texture :name textures-node :names)
-       (g/connect textures-node :name-counts texture :name-counts)
-       (g/connect self :samplers texture :samplers)
-       (g/connect self :default-tex-params texture :default-tex-params))))))
+     (g/connect texture :_node-id self :nodes)
+     (g/connect texture :texture-gpu-textures self :texture-gpu-textures)
+     (g/connect texture :texture-infos self :texture-infos)
+     (when (not internal?)
+       (concat
+         (g/connect texture :texture-names self :texture-names)
+         (g/connect texture :dep-build-targets self :dep-build-targets)
+         (g/connect texture :pb-msg self :texture-msgs)
+         (g/connect texture :build-errors textures-node :build-errors)
+         (g/connect texture :node-outline textures-node :child-outlines)
+         (g/connect texture :name textures-node :names)
+         (g/connect textures-node :name-counts texture :name-counts)
+         (g/connect self :samplers texture :samplers)
+         (g/connect self :default-tex-params texture :default-tex-params))))))
```

Comment:
Indentation, but also note I've renamed the `own-` prefix from the `GuiSceneNode` inputs where gui resources attach. Them belonging to the scene is now implied, and instead of having `own-` and `aux-` variants of each, we have `own-basic-gui-scene-info` and `aux-basic-gui-scene-info`, as well as counterparts for `costly-gui-scene-info`.